salt/salt.spec

```diff
@@ -632,12 +632,15 @@ Requires(pre):  %fillup_prereq
 Salt ssh is a master running without zmq.
 it enables the management of minions over a ssh connection.
 
-%package tests
+%package -n python3-salt-testsuite
 Summary:        Unit and integration tests for Salt
 Requires:       %{name} = %{version}-%{release}
+Requires:       python3-pytest
+Requires:       python3-pytest-salt-factories
```

Comment:
I think we need to add version specifiers for these two.

Reply:
About the files placement, I just aligned them with the way how the test file are placed inside `python3xx-testsuite` subpackage of `python3xx` package: https://build.opensuse.org/package/view_file/SUSE:SLE-15-SP5:Update/python310/python310.spec?expand=1
```
%files -n %{python_pkg_name}-testsuite
%defattr(644, root, root, 755)
%{sitedir}/test
%{sitedir}/*/test
%{sitedir}/*/tests
...
```